facia/app/conf/HealthCheck.scala

```diff
@@ -3,8 +3,12 @@ package conf
 import model.NoCache
 import play.api.mvc.Action
 
-object HealthCheck extends AllGoodHealthcheckController(9008, "/uk") {
-  def cdnHealthcheck = Action{ request =>
-    NoCache(if (isOk) Ok("OK") else ServiceUnavailable("Service Unavailable"))
-  }
+object HealthCheckController extends CachedHealthCheckController {
+  override val paths = Seq("/uk/business")
+  override val testPort = 9008
```

Comment:
Is it possible not to hardcode this? Is this helpful: http://stackoverflow.com/a/21353658 ?